### PR TITLE
Boilerplate

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
@rudigiesler @kaitcrawford ready for review

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+casepro
```
Comment (rudigiesler) [inline requirements.txt]: I don't think that casepro is on PyPI, we might have to use a git link here.

**Comment by rudigiesler** [comment]
Just the one comment from me. Maybe we should also add a travis file, even if all it does is run `pip install`.

**Comment by justinvdm** [comment]
@rudigiesler I've removed casepro as a dependency for now, since it is not currently installable, and since this plugin doesn't have any tests. We can change that once it is installable. Maybe that is enough for now?

**Comment by rudigiesler** [comment]
:+1:

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      long_description=readme,
+      classifiers=[
+          "Programming Language :: Python",
+          "Framework :: Django",
+          "Topic :: Internet :: WWW/HTTP",
+          "Topic :: Internet :: WWW/HTTP :: WSGI :: Application",
+      ],
+      author='Praekelt Foundation',
+      author_email='dev@praekelt.com',
+      url='http://github.com/praekelt/molo.usermetadata',
```
Comment (KaitCrawford) [inline setup.py:24]: @justinvdm is this url correct?

**Pull request merged**